src/helpers/OTSessionHelper.js

```diff
@@ -43,7 +43,7 @@ const sanitizeSessionEvents = (events) => {
       streamPropertyChanged: 'onStreamPropertyChanged',
     },
   };
-  return reassignEvents('session', customEvents, events);
+  return reassignEvents('session', customEvents, events, undefined, sessionId);
```

Comment:
Hmm, is there an alternative approach we can take instead of passing in undefined as an argument? Maybe we can refactor `reassignEvents` or create one specifically for sessions?

Reply:
Just refactored it a bit, wdyt?